spec/API.test.js

```diff
@@ -6,18 +6,19 @@ require('babel-polyfill');
 
 describe('API tests', () => {
   it('Should get a response given a good GET request', async (done) => {
-    // const response = await request(app).get('/api/adflhie')
-    // expect(response.statusCode).toBe(500)
+    await request(app)
+      .get('/api/001')
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+      });
     done();
   }, 90000);
 
   it('Should 500 a bad GET request', async (done) => {
-    // const response = await request(app).get('/api/adflhie')
-    // expect(response.statusCode).toBe(500)
     await request(app)
-      .get('/api/001')
+      .get('/api/adflhie')
```

Comment:
this is supposed to be an invalid id right? maybe change 'adflhie' to 'invalid id' to make that clearer.